docs/asyncapi.yaml

```diff
@@ -338,19 +338,19 @@ components:
         properties:
           invoiceNumber:
             type: string
-            description: Unique request for payment
+            description: Unique request for payment, required
           frn:
             type: number
-            description: Firm Reference Number
+            description: Firm Reference Number must be 10 digits long, required
           success:
             type: boolean
-            description: Whether request was successfully loaded by DAX
+            description: Whether request was successfully loaded by DAX, required
           acknowledged:
             type: string
-            description: Date acknowledged in DAX
+            description: Date acknowledged in DAX, required
```

Comment:
Does this best describe this?

It's a string date, i.e. Fri Jan 21 2022 00:00:00 GMT+0000 (Greenwich Mean Time) which you can get from `new Date().toString()`

However it is successfully converted by `Sequelize` to `DataTypes.DATE` in `app/data/models/completed-payment-request.js`

Reply:
This is fine.

To be honest, I'd rather responses was sending dates in an ISO format rather than what it is sending and we will look to change that.  You're right that sequelize will handle both date formats so let's not make this too restrictive.